mlflow/gateway/providers/ai21labs.py

```diff
@@ -21,7 +23,7 @@ async def completions(self, payload: completions.RequestPayload) -> completions.
         self.check_for_model_field(payload)
         key_mapping = {
```

Comment:
No temperature scaling to be done for AI21Labs. Range is -Inf to -Inf for AI21. We can stick with 0-2 restriction for now (it's currently 0-1) and get feedback.